### ISSUE TITLE
[Adaptive Cards-Designer]: Test Editor area is not adopting high contrast mode in the Designer page. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8882]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test environment:**  
OS: Win11  
OS Version: 24H2 (OS Build 26100.1)    
URL:[Adaptive Cards](https://adaptivecards.io/) 
Edge dev Browser version: 125.0.2506.0 (Official build) dev (64-bit)  
Screen reader: Narrator. 
Mode: HC mode 

**Prerequisites:**   
Turn on the High contrast mode form System settings > Accessibility > contrast theme   

**Repro Steps:** 
1. Open URL in the Edge browser & sign in with valid credentials. 
2. It will open the Adaptive cards home page 
3.Activate the designer control and observe the issue on the text edit area in HC mode 

**Actual Result:**  
In the designer page, the editor page is not adapting to the high contrast settings mode. 

**Expected Result:** 
The text edit area should adapt to the high contrast settings mode, so that low vision users can access the page. 

**Observation:**
1. Expanded control icon also not visible properly on HC mode on the designer page. 
2. Close button is not properly visible on HC mode [Path: Designer tab> create a new card] 

**User Experience:**  
The low vision users will be impacted if the page is not adapting to the high contrast settings, due to which low vision users will not be able to navigate the page. 

MAS Reference: 
[MAS 4.3.1 – No Disruption of Accessibility Features](https://aka.ms/MAS4.3.1)

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

<img width="960" alt="Text edit area is not adopting the HC mode" src="https://github.com/microsoft/AdaptiveCards/assets/95913748/4889531f-86e7-48eb-9c7c-76e5741c0553">

<img width="958" alt="Observation2_Close icon  is not visible in HC mode" src="https://github.com/microsoft/AdaptiveCards/assets/95913748/81be814a-5a86-43a4-9ff0-a88b00f5c3b7">

<img width="960" alt="Observation1_Dropdown icon is not visible in HC mode" src="https://github.com/microsoft/AdaptiveCards/assets/95913748/632bac36-956f-4fd2-84f0-428a77ed8784">

### Card JSON

```JSON
NA
```
